huaweicloud/services/acceptance/cce/resource_huaweicloud_cce_cluster_test.go

```diff
@@ -537,3 +577,59 @@ resource "huaweicloud_cce_cluster" "test" {
 }
 `, common.TestVpc(rName), rName)
 }
+
+func testAccCluster_secGroup(rName string) string {
+	return fmt.Sprintf(`
+%s
+
+resource "huaweicloud_networking_secgroup" "test1" {
+  name = "%[2]s-secgroup-1"
```

Comment:
do you need to add some required rules here to make the cluster work ok?

Reply:
the secgroup only used when create worker nodes, we don't need to add rules in this test case